src/pretix/base/services/orders.py

```diff
@@ -2089,7 +2089,7 @@ def _try_auto_refund(order, manual_refund=False, allow_partial=False, source=Ord
     else:
         proposals = order.propose_auto_refunds(refund_amount)
         can_auto_refund_sum = sum(proposals.values())
-        can_auto_refund = (allow_partial and can_auto_refund_sum) or can_auto_refund_sum == refund_amount
+        can_auto_refund = can_auto_refund_sum if allow_partial else can_auto_refund_sum == refund_amount
```

Comment:
After we figured out the other one caused a problem I'd very much prefer to not have this modification as well. It's not the same thing.